src/toady/models.py

```
@@ -0,0 +1,161 @@
+"""Data models for GitHub review threads and comments."""
+
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import Any, Dict, List, Union
+
+
+@dataclass
+class ReviewThread:
+    """Represents a GitHub pull request review thread.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)